src/components/media-loader.js

```diff
@@ -195,7 +195,8 @@ AFRAME.registerComponent("media-loader", {
         this.el.setAttribute("gltf-model-plus", {
           src: accessibleUrl,
           contentType: contentType,
-          inflate: true
+          inflate: true,
+          scale: 0.0001
```

Comment:
So.. its a bit weird that we are going to have this scale property dangling around that actually doesnt reflect the current scale of anything. Maybe just renaming it to `initialScale` or something would help? but still feels pretty weird that we have this but then also modify the meshes scale later to reverse this.

Reply:
Yeah... to be honest, using this scale property to fix this seems like a bit of a hack. Can we do this scaling when the model is loaded but before it's added to the scene? I don't like adding more events, but I also don't really like pushing the complexity into the schema when there is already a scale component.

Reply:
we perform the initial sizing and scale in response to the `media-loaded` event, which happens > 1 frame after the gltf model has been spawned into the scene. if there's a straightforward way to resolve this issue that is cleaner than this approach then that sounds good to me, but my assumption was that this PR I wrote was the best way to do it given the way we deal with media

Reply:
i'm going to rename this to `modelToWorldScale` and push it tmw unless there are objections or suggestions on a cleaner solution